docs/pages/blog/base-ui-2024-plans.md

```diff
@@ -0,0 +1,105 @@
+---
+title: An exciting year ahead for Base UI
+description: The unstyled component library will get a stable release, lots of new components, and even better DX in 2024.
+date: 2024-02-09T00:00:00.000Z
+authors: ['michaldudak', 'danilo-leal']
```

Comment:
This feels inconsistent, most of the content was written by danilo-leal. So either it's the actual author, and would probably more accurate:
```suggestion
authors: ['danilo-leal', 'michaldudak']
```

or it's about the team working on it / involved in the decisions, hence:

```suggestion
authors: ['michaldudak', 'danilo-leal', 'colmtuite', 'oliviertassinari']
```

sounds more accurate